test/unit/test_dwarf.c

```diff
@@ -82,6 +82,7 @@ bool test_dwarf3_c_basic(void) { // this should work for dwarf2 aswell
 	rz_io_bind(io, &bin->iob);
 
 	RzBinOptions opt = { 0 };
+	opt.obj_opts.elf_load_sections = true;
```

Comment:
I see you have to do this all the time, so maybe it would make sense to have a variable that express the opposite of this, so it is false by default. Like... `elf_noload_sections`. WDYT?

Reply:
Negative sentence variable is a code smell. I don't know if those could be rewrite with rz_bin_init_options (eld_load_sections will be set to true in this function)

Reply:
right, better to set it to true directly in rz_bin_init_options.